docs/swagger.json

```diff
@@ -673,6 +673,17 @@
         "operationId" : "segments_getSegments",
         "produces" : [ "application/json" ],
         "parameters" : [ {
+          "name" : "includeType",
+          "in" : "query",
+          "description" : "Include additional segments not owned by user. The \"all\" option takes precedence over \"shared\"",
+          "required" : false,
+          "type" : "array",
+          "items" : {
+            "type" : "string",
+            "enum" : [ "shared", "all", "templates", "deleted", "internal", "curatedItem" ]
```

Comment:
Do we want to show all the include types or just a subset of these?
Also, we should expose this parameter for calculated metrics at the same time.